layouts/index.html

```diff
@@ -310,7 +353,7 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Fully supported, always free" }}</h4>
                         <div>
                             <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
                             <p>
-                                {{ i18n "Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy" }} - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}.</a>
+                                Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS. Switching is easy  - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}</a>.
```

Comment:
Any reason localization was removed from this paragraph? If not please re-add it before I merge it in (or if you like I can).

Reply:
I have no idea, we need to ask @sakelestemur if that was intentional.  I did check and this how it looks in the archive he provided me with.  I think that this was out of scope of the changes he was doing.

Reply:
I'm guessing he didn't pull in the latest changes so it was just missed. You can go ahead and re-wrap that string with the `{{ i18n " " }}` tags and then we can merge everything.

Reply:
Done.